cni/deployments/kubernetes/Dockerfile.install-cni

```diff
@@ -1,15 +1,17 @@
-FROM ubuntu:xenial
+# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
+ARG BASE_DISTRIBUTION=default
 
-LABEL description="Istio CNI plugin installer."
+# Version is the base image version from the TLD Makefile
+ARG BASE_VERSION=latest
+
+# The following section is used as base image if BASE_DISTRIBUTION=default
+FROM docker.io/istio/base:${BASE_VERSION} as default
 
-RUN apt-get update && apt-get install -qqy --no-install-recommends \
-jq='1.5*' \
-&& rm -rf /var/lib/apt/lists/*
+LABEL description="Istio CNI plugin installer."
 
 COPY istio-cni /opt/cni/bin/
 COPY istio-iptables /opt/cni/bin/
 COPY install-cni /usr/local/bin/
-COPY istio-cni.conf.default /istio-cni.conf.tmp
```

Comment:
This file is no longer used, so I removed it. This was originally used as the default template if the user doesn't set the `CNI_NETWORK_CONFIG` or `CNI_NETWORK_CONFIG_FILE` variables: [here](https://github.com/istio/cni/blame/8bdd80240a8f7286420c72da6542f516260d86d7/deployments/kubernetes/install/scripts/install-cni.sh#L161).

But I think in the process of refactoring, that file's use got removed: [here](https://github.com/istio/cni/blame/master/deployments/kubernetes/install/scripts/install-cni.sh#L184). This could have introduced a bug if none of the variables were set, but `CNI_NETWORK_CONFIG` is set in the helm charts by default.

After rewriting the shell script to Golang, at least one of the two variables must be set, else it will exit with error: [here](https://github.com/istio/istio/blob/ed115aca2eef87af55b2b1ed7953e55fd8e8e3eb/cni/pkg/install-cni/pkg/install/cniconfig.go#L108).